src/components/campaigns/ViewCampaignProgressCard.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { useTranslation } from 'next-i18next'
+import { routes } from 'common/routes'
+import CampaignProgress from './CampaignProgress'
+import LinkButton from 'components/common/LinkButton'
+import { Box, Button, Grid, createStyles, makeStyles, Theme } from '@material-ui/core'
+import { Share, Favorite } from '@material-ui/icons'
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    outlinedButton: {
+      color: theme.palette.primary.dark,
+      border: `2px solid ${theme.palette.primary.dark}`,
+      borderRadius: theme.spacing(3),
+      width: '100%',
+      whiteSpace: 'nowrap',
+      height: theme.spacing(6),
+      marginTop: theme.spacing(3),
+      fontSize: theme.typography.pxToRem(15),
+      fontWeight: 500,
+    },
+    donateNowButton: {
+      color: theme.palette.common.white,
+      backgroundColor: theme.palette.primary.main,
+      border: `2px solid ${theme.palette.primary.main}`,
+      borderRadius: theme.spacing(3),
+      width: '100%',
+      height: theme.spacing(6),
+      marginTop: theme.spacing(3),
+      fontSize: theme.typography.pxToRem(15),
+      fontWeight: 500,
+    },
+    buttonWrapper: {
+      display: 'inline-flex',
+      width: '100%',
+      gap: theme.spacing(2),
+    },
+  }),
+)
+
+export default function ViewCampaignProgressCard() {
```

Comment:
In next iteration we can move this file in dedicated folder "view campaign" as more components will appear I imagine